locales/base/translation.json

```diff
@@ -1064,6 +1064,24 @@
     "unsupportedChain": {
       "title": "Unsupported chain",
       "description": "{{dappName}} is requesting this action on a chain not supported by {{appName}}. Please make sure Celo is the active chain on the dapp side."
+    },
+    "session": {
+      "failUnsupportedNamespace": {
+        "title": "Unsupported chain",
+        "description": "{{dappName}} would like to connect with a chain not supported by {{appName}}. Please make sure Celo is selected on the dapp side."
+      },
+      "warnUnsupportedChains": {
+        "title": "Unsupported chain",
+        "description": "{{dappName}} didn't not specify Celo as a supported chain. Some features may not work as expected."
+      },
+      "warnSomeUnsupportedMethods": {
+        "title": "Unsupported methods",
+        "description": "{{dappName}} specified some methods that are not supported by {{appName}}. Some features may not work as expected."
+      },
+      "warnSomeUnsupportedEvents": {
+        "title": "Unsupported events",
+        "description": "{{dappName}} specified some events that are not supported by {{appName}}. Some features may not work as expected."
```

Comment:
Yes I agree, I also wanted devs to get a bit more details, so I landed with this solution which isn't entirely speaking to normal users. Ideally there would be a button to see those technical details, and we could keep the message more generic.
Are you still fine merging this as is? Trying to avoid more changes on this one. Most users won't see those warnings anyway.

Reply:
yes i don't have strong feelings about it :)